packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1780,6 +1800,14 @@ contract OPContractsManager is ISemver {
         Claim absolutePrestate;
     }
 
+    /// @notice The input required to identify a chain for updating prestates
+    struct UpdatePrestateInput {
+        ISystemConfig systemConfigProxy;
+        IProxyAdmin proxyAdmin;
```

Comment:
We can remove the `proxyAdmin` arg, since you can call `systemConfigProxy.proxyAdmin()` to get it's proxy admin

Reply:
This is fixed. We could also remove `proxyAdmin` from `AddGameInput` for the same reason. I'll deal with that in another PR.